#include "server.h"

::grpc::Status CppAImpl::Cpp_A_1(::grpc::ServerContext *context,
                                 const ::protos::CommonMessage *request,
                                 ::protos::CommonMessage *response) {}

::grpc::Status CppAImpl::Cpp_A_2(::grpc::ServerContext *context,
                                 const ::protos::CommonMessage *request,
                                 ::protos::CommonMessage *response) {}

::grpc::Status CppAImpl::Cpp_A_3(::grpc::ServerContext *context,
                                 const ::protos::CommonMessage *request,
                                 ::protos::CommonMessage *response) {}

int main(void) { return 0; }
